sqlmesh/dbt/target.py

```diff
@@ -42,6 +42,14 @@
     t.Type[IncrementalUnmanagedKind],
 ]
 
+# We only serialize a subset of fields in order to avoid persisting sensitive information
+SERIALIZABLE_FIELDS = {
+    "type",
+    "name",
+    "database",
+    "schema_",
```

Comment:
Is this field serialized as `schema_` or `schema`. It needs to be the latter.

Reply:
Unit test confirms its the latter.

Reply:
lol was wondering the same